aspnetcore/fundamentals/servers/kestrel/endpoints.md

```diff
@@ -199,7 +200,27 @@ In the following *appsettings.json* example:
 }
 ```
 
-An alternative to using `Path` and `Password` for any certificate node is to specify the certificate using certificate store fields. For example, the `Certificates` > `Default` certificate can be specified as:
+Schema notes:
+
+* Endpoints names are case-insensitive. For example, `HTTPS` and `Https` are valid.
```

Comment:
Maybe link to https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-5.0#configuration-keys-and-values which states "Configuration keys: Are case-insensitive. For example, ConnectionString and connectionstring are treated as equivalent keys."